browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -1140,6 +1177,8 @@ void BraveBrowserView::OnActiveTabChanged(content::WebContents* old_contents,
   if (supports_split_view) {
     UpdateSplitViewSizeDelta(old_contents, new_contents);
 
+    UpdateSplitViewOrientation();
```

Comment:
After creating split view from tab context menu,
the orientation for newly created tile is not vertical, It's previously used orientation.
Maybe the reason is active tab change is not happened for tile creating?
Also it would be great to have some tests for this orientation mode.

Reply:
Nice catch! Fixed and added test